tests/fixups/TraceFixup/Logging.h

```diff
@@ -129,16 +130,83 @@ inline void LogString(const char* name, const wchar_t* value)
 {
     Log("\t%s=%ls\n", name, value);
 }
+inline std::string InterpretStringA(const char* value)
+{
+	return value;
+}
+inline std::string InterpretStringA(const wchar_t* value)
+{
+	return narrow(value);
+}
 
 inline void LogCountedString(const char* name, const char* value, std::size_t length)
 {
     Log("\t%s=%.*s\n", name, length, value);
 }
+inline std::string InterpretCountedString(const char* name, const char* value, std::size_t length)
+{

```

Comment:
can we consolidate the Log* and Interpret* function pairs?  I might start by renaming Interpret* to Format* for clarity, then each Log* could simply OuptutDebugString(Format*(...));

Reply:
In general, I had a concern that I did not want to affect the output of Log* functions, as certain third party tool vendors might be depending on the exact output of the existing debug strings. Some of those strings are actually incorrect.  This case is an example of attempted consistency to always have a separate function.  I can do some consolidation, but still want to leave the output of Log* untouched when different.